interface-definitions/service_ipoe-server.xml.in

```diff
@@ -70,6 +70,18 @@
                           <constraintErrorMessage>VLAN IDs need to be in range 1-4094</constraintErrorMessage>
                         </properties>
                       </leafNode>
+                      <leafNode name="static-ip">
+                        <properties>
+                          <help>Static client IP address</help>
+                          <valueHelp>
+                            <format>ipv4</format>
+                            <description>IPv4 address</description>
```

Comment:
Is there a reason not to add a static IPv6 option right away? Or it's not supported by accel-ppp?

Reply:
I couldn't find in the documentation whether it is possible to assign a static ipv6 address. When manually adding the ipv6 address to the ipoe.chap-secrets file, it was not assigned to the client. Now I'm asking Sviridya if this is possible at all, or if only the static ip address can be configured

Reply:
Could you try this?
```
username3     *            password789      192.0.2.100,[2001:db8::2]
```
Pay attention it probably requires an additional option:
```
set system sysctl parameter net.ipv6.conf.default.addr_gen_mode value '0'
```

Reply:
It didn't work
```
Dec 16 09:51:28 systemd[1]: Starting accel-ppp@ipoe.service - Accel-PPP - High performance VPN server application for Linux...
Dec 16 09:51:28 systemd[1]: Started accel-ppp@ipoe.service - Accel-PPP - High performance VPN server application for Linux.
Dec 16 09:51:39 accel-ipoe[6649]: ipoe0:eth1: eth1: authentication succeeded
Dec 16 09:51:39 accel-ipoe[6649]: ipoe0:eth1: no free IPv4 address
Dec 16 09:51:39 accel-ipoe[6649]: ipoe0:eth1: ipoe: session finished
Dec 16 09:51:42 accel-ipoe[6649]: ipoe0:eth1: eth1: authentication succeeded
Dec 16 09:51:42 accel-ipoe[6649]: ipoe0:eth1: no free IPv4 address
Dec 16 09:51:42 accel-ipoe[6649]: ipoe0:eth1: ipoe: session finished
Dec 16 09:51:45 accel-ipoe[6649]: ipoe0:eth1: eth1: authentication succeeded
Dec 16 09:51:45 accel-ipoe[6649]: ipoe0:eth1: no free IPv4 address
Dec 16 09:51:45 accel-ipoe[6649]: ipoe0:eth1: ipoe: session finished
```

with only ipv4 addres everything works fine, but with ipv6 it doesn't